Services/CO.CDP.Person.WebApi/Api/Person.cs

```diff
@@ -22,12 +22,21 @@ public static class EndpointExtensions
     public static void UsePersonEndpoints(this WebApplication app)
     {
         app.MapPost("/persons", async (RegisterPerson command, IUseCase<RegisterPerson, Model.Person> useCase) =>
-            await useCase.Execute(command)
-                .AndThen(person =>
-                    Results.Created(new Uri($"/persons/{person.Id}", UriKind.Relative), person))
-                )
+        {
+            if (command == null)
```

Comment:
command will never be null here from ASP.NET pipeline

Reply:
I have removed that command check now.